mesonbuild/interpreter.py

```diff
@@ -2879,7 +2906,7 @@ def add_find_program_override(self, name, exe):
 
     # TODO update modules to always pass `for_machine`. It is bad-form to assume
     # the host machine.
-    def find_program_impl(self, args, for_machine: MachineChoice = MachineChoice.HOST, required=True, silent=True):
+    def find_program_impl(self, args, for_machine: MachineChoice = MachineChoice.HOST, required=True, silent=True, wanted=[], version_regex=''):
```

Comment:
__[Flake8]__ [B006] Do not use mutable data structures for argument defaults. All calls reuse one instance of that data structure, persisting changes between them. ([link](https://sider.review/gh/repos/19784232/pulls/5637#issue-4326071))

<sub>posted by Sider</sub>